packages/amplify-category-auth/resources/cloudformation-templates/auth-template.yml.ejs

```diff
@@ -43,6 +43,7 @@ Parameters:
 
 Conditions:
   ShouldNotCreateEnvResources: !Equals [ !Ref env, NONE ]
+  HasHostedUIProviderCreds: !Not [ !Equals [ !Ref hostedUIProviderCreds, NONE ]]
```

Comment:
You'd need to set the default value to NONE in the template itself I think or are you dynamically passing this value during deploy time as an input param?

Reply:
I am dynamically deploying this value if it exists

Reply:
I am not entirely sure if `NONE` can be added as a default value 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html